pkg/validator/schema.go

```diff
@@ -290,6 +310,20 @@ func applySchemaCheck(conf *config.Configuration, checkID string, test schemaTes
 	if err != nil {
 		return nil, err
 	}
+	for kind := range check.AdditionalValidators {
+		if !passes {
+			break
+		}
+		resources := test.ResourceProvider.Resources[kind]
+		objects := make([]interface{}, len(resources))
```

Comment:
We already have funk here, you could do this with a `Map`

Reply:
duly funked